src/components/Message/Message.style.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * @file Message.style.js
+ * Exports Message component styles.
+ */
+
+import red from '@material-ui/core/colors/red';
+
+export default theme => ({
+  message: {
+    marginTop: theme.spacing.unit
+  },
+  error: {
+    color: red[500]
+  },
+  info: {
+    color: 'rgba(255, 255, 255, 0.7)'
```

Comment:
The default color for info text in MaterialUI forms isn't exposed anywhere in the theme. It's just a slightly opaque white, which means it sort of inherits a bit of color from whatever it's background color is. That's why I'm not pulling this from an object within `@material-ui/core`.